src/common/options/osd.yaml.in

```diff
@@ -739,6 +739,14 @@ options:
   default: 32
   flags:
   - startup
+- name: osd_log_slow_op_to_clog
```

Comment:
please reconsider the flag name. Possibly "log_every_slow_op" or "aggregated_slow_ops_logging"? otherwise - it is not clear what are the options selected by the flag.

Reply:
Replaced option `osd_log_slow_op_to_clog` with `osd_aggregated_slow_ops_logging`.